src/components/date-picker/flatpickr-adapter/flatpickr-adapter.scss

```diff
@@ -94,6 +95,33 @@ svg {
                 background-color: transparent;
                 color: var(--mdc-theme-primary);
             }
+            &:before {
+                top: -60%;
+                opacity: 1;
+            }
+        }
+
+        &:before {
```

Comment:
Fixed! 😊

Reply:
🤩 this looks very nasty!

Reply:
just discovered that we have a little UI issue here. 
<img width="179" alt="image" src="https://user-images.githubusercontent.com/35954987/199022523-128e7678-db15-4cda-813c-20018bf75b74.png">

If the label is too long, and the day is close to the edge, it'd be cut off.

Maybe we can fix it with some `z-index` or something. I'll get back to this later, in case you have not already managed to fix it.. I'm working on another issue right now.

Reply:
Yeah, I saw those too, but I know better than than voluntarily entering a fight with flatpickr, so I'm happy to leave it to you 😜

Reply:
- 😄 I actually think this is perfectly fine to leave for review. I'll fix those later in another issue.

Reply:
I agree!